.travis.yml

```diff
@@ -10,18 +10,14 @@ rvm:
 sudo: false
 
 env:
-  - PUPPET_GEM_VERSION='~> 4.7.0' COVERAGE=yes
-  - PUPPET_GEM_VERSION='~> 2.7.0'
-  - PUPPET_GEM_VERSION='~> 3.7.0'
-  - PUPPET_GEM_VERSION='~> 3.8.0'
-  - PUPPET_GEM_VERSION='~> 4.0.0'
-  - PUPPET_GEM_VERSION='~> 4.1.0'
-  - PUPPET_GEM_VERSION='~> 4.2.0'
-  - PUPPET_GEM_VERSION='~> 4.3.0'
+  - PUPPET_GEM_VERSION='~> 4.8.0' COVERAGE=yes
```

Comment:
Please add one entry with '~> 4.0' to always test against the latest released version.